core/app/models/spree/adjustment.rb

```diff
@@ -73,10 +73,6 @@ class Adjustment < Spree::Base
     extend DisplayMoney
     money_methods :amount
 
-    def closed?
```

Comment:
@siddharth28 - We are using this method here - https://github.com/siddharth28/spree/blob/master/core/app/models/spree/adjustment.rb#L88.